packages/legend-query-builder/src/stores/milestoning/QueryBuilderMilestoningState.ts

```diff
@@ -165,6 +151,23 @@ export class QueryBuilderMilestoningState implements Hashable {
     return milestoningParameter;
   }
 
+  getMilestoningBuilderHelper(
```

Comment:
see my comment above, I believe we can chuck this inside of `QueryBuilderMilestoningImplementation`